package.json

```diff
@@ -23,6 +23,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.6.0",
+    "react-number-format": "^5.1.1",
```

Comment:
previously commented https://github.com/WavesHQ/metascan/pull/35#discussion_r1021109999
let's use `<NumericFormat>`

Reply:
updated to use `import NumericFormat from "@components/commons/NumericFormat";`